commcare_connect/opportunity/tw_forms.py

```diff
@@ -0,0 +1,153 @@
+from crispy_forms.helper import FormHelper
+from crispy_forms.layout import HTML, Column, Field, Fieldset, Layout, Row
+from django import forms
+from django.core.exceptions import ValidationError
+
+from .models import DeliverUnit, DeliverUnitFlagRules, FormJsonValidationRules, OpportunityVerificationFlags
+
+BASE_INPUT_CLASS = "w-full p-2 border border-gray-300 rounded-lg focus:ring-2 focus:ring-blue-500"
+SELECT_CLASS = "w-full p-2 border border-gray-300 rounded-lg"
+TEXTAREA_CLASS = "w-full p-2 border border-gray-300 rounded-lg"
+CHECKBOX_CLASS = "form-checkbox text-blue-600 rounded"
+
+
+class OpportunityVerificationFlagsConfigForm(forms.ModelForm):
+    class Meta:
+        model = OpportunityVerificationFlags
+        fields = (
+            "duplicate",
+            "gps",
+            "location",
+            "form_submission_start",
+            "form_submission_end",
+            "catchment_areas",
+        )
+        widgets = {
+            "form_submission_start": forms.TimeInput(attrs={"type": "time", "class": BASE_INPUT_CLASS}),
+            "form_submission_end": forms.TimeInput(attrs={"type": "time", "class": BASE_INPUT_CLASS}),
+            "location": forms.NumberInput(attrs={"class": BASE_INPUT_CLASS}),
+        }
+        labels = {
+            "duplicate": "Check Duplicates",
+            "gps": "Check GPS",
+            "form_submission_start": "Start Time",
+            "form_submission_end": "End Time",
+            "location": "Location Distance",
+            "catchment_areas": "Catchment Area",
+        }
+        help_texts = {
+            "location": "Minimum distance between form locations (metres)",
+            "duplicate": "Flag duplicate form submissions for an entity.",
+            "gps": "Flag forms with no location information.",
+            "catchment_areas": "Flag forms outside a user's assigned catchment area",
+        }
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        self.fields["duplicate"].widget.attrs["class"] = CHECKBOX_CLASS
```

Comment:
@black All you need to do to be able to customize the styling is by applying various tailwind classes on the fields here in Python. You can also change non-field related styles directly by editing `verification_flags_config.html` file https://github.com/dimagi/commcare-connect/pull/543/files#diff-584b1126b36c4ea38e26c452e1b8b70b8f8775a15233cf7ca0fcc348be27a112